GETTING_STARTED.md

```diff
@@ -75,7 +75,7 @@ Then the tests can be run with:
     
 Note: If your OS is not explicitly supported in the setup guides then not all tests may pass. However, you may still be able to develop. Get in touch with the [#dev][slack-dev] channel on Slack to troubleshoot issues and determine if they will preclude you from contributing to OFN.
 
-Additionally, make sure that the time zone on your machine matches the one defined in `config/application.yml`. If there is a discrepancy, it may cause some RSpec tests to fail.
+Additionally, make sure that the time zone on your machine matches the one defined in `config/application.yml` (it should be Melbourne). Setting your machine's time zone to any other zone has the potential to cause test failures.
```

Comment:
I think the part about test failures got solved in https://github.com/openfoodfoundation/openfoodnetwork/pull/2948. Can you verify that and take it out if solved?

Reply:
Done!